pkg/openstack/tasks/networks.go

```diff
@@ -0,0 +1,223 @@
+// SPDX-FileCopyrightText: 2025 SAP SE or an SAP affiliate company and Gardener contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package tasks
+
+import (
+	"context"
+	"encoding/json"
+
+	asynqclient "github.com/gardener/inventory/pkg/clients/asynq"
```

Comment:
Imports don't seem to be ordered according to `goimports{-reviser}`.

Please run `make goimports goimports-reviser`